releasenotes/README.md

```diff
@@ -155,83 +35,94 @@ Run `sfdx plugins --core` to display the version of Salesforce CLI and the insta
+### New Parameters
 
-### Removed Commands
+* `force:mdapi:deploy --soapdeploy`
 
-These commands were deprecated in v49 and removed in v50.  You can continue using them by installing the [Lightning Testing Service CLI Plug-in](https://github.com/forcedotcom/plugin-lightning-test-service). Salesforce doesn't support or update this plug-in.
+  The `soapdeploy` parameter will cause the metadata deploy to use the SOAP API, instead of the REST API.
 
-* `force:lightning:test:install`
-* `force:lightning:test:run`
+* `force:package:create --orgdependent`
 
-### New Parameters
+  For unlocked packages only, allows the package to depend on unpackaged metadata in the installation org.
+
+### Deprecated Parameters
+
+* `force:org:shape:create`
+
+  The `definitionfile` parameter has been removed from this command. This parameter is part of the Org Shape feature which is currently in beta and has been removed based on feedback that users prefer storing Org Shape in the shape source org instead of a scratch definition file.
+
+### Second Generation Packaging
 
-* `force:package:create --errornotificationusername (-o)`
-* `force:package:update --errornotificationusername (-o)`
+* The release version stamped on a package version is now available in the response of command `force:package:version:report` when the user specifies the `--verbose` parameter.
 
-    Designate an active Dev Hub org user to receive email notifications for unhandled Apex exceptions, as well as install, upgrade, or uninstall failures associated with your packages. Specify the username during package creation, or when you update the package with the `force:package:update` command.
+* Package version CLI commands are enhanced to support component deletion
+  * If force:package:version:create completes successfully and if any metadata has been removed from the package version in comparison to its ancestor, a generic warning message is shown in the `force:package:version:create` output highlighting that some metadata was removed.
+  * Command `force:package:version:list` displays this info (boolean, whether or not metadata was removed) when used with --verbose param
+  * Command `force:package:version:report` displays this info (boolean, whether or not metadata was removed).
+  * Command `force:package:version:promote` displays a warning if the version being promoted contains removed components and if --noprompt is not specified.
 
 ### Other Changes
 
-* When parsing the `.forceignore` file, Salesforce CLI uses the same rules and patterns as [git uses with the `.gitignore` file](https://git-scm.com/docs/gitignore). As a result, some entries in your existing `.forceignore` files might be invalid. The `force:source:*` commands warn you about any invalid entries.  
+* FIX: The `force:user:password:generate` no longer fails whe run against Spring '21 release of Salesforce.
```

Comment:
Edit: "The `force:user:password:generate` command no longer fails when run against the Spring '21 Salesforce release."